src/views/send-solana/index.tsx

```diff
@@ -0,0 +1,91 @@
+import { TOKEN_PROGRAM_ID } from "@solana/spl-token";
+import { useConnection, useWallet } from "@solana/wallet-adapter-react";
+import {
+  LAMPORTS_PER_SOL,
+  SystemProgram,
+  PublicKey,
+  Transaction,
+} from "@solana/web3.js";
+import { Button, Input, Spin } from "antd";
+import React, { FC, useState } from "react";
+import { LABELS } from "../../constants";
+import { notify } from "../../utils/notifications";
+
+export const SendSolana: FC = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [amount, setAmount] = useState(0);
+  const [destination, setDestination] = useState("");
+  const { publicKey, sendTransaction } = useWallet();
+  const { connection } = useConnection();
+
+  const handleFormSubmit: React.FormEventHandler<HTMLFormElement> = async (
+    e
+  ) => {
+    try {
+      e.preventDefault();
+      setIsLoading(true);
+      const lamports = LAMPORTS_PER_SOL / amount;
+
+      // Note this transaction instruction is formed internally by the SystemProgram class;
+      // however, most program calls look similar to this.
+      const transactionInstruction = SystemProgram.transfer({
+        fromPubkey: publicKey!,
+        toPubkey: new PublicKey(destination),
+        lamports,
+        programId: TOKEN_PROGRAM_ID,
```

Comment:
Since this is a native lamport transfer, should this be passing the token program? This is done in a few places in the PR.